### PR TITLE
Comet.ml logger - add usage tracking

### PR DESCRIPTION
**Pull request by alexkuzmik**
## What does this PR do?

Fixes https://github.com/Lightning-AI/lightning/issues/14905

### CODE REVIEW

**Review by awaelchli — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

**Review by otaj — APPROVED** [review]

File: src/pytorch_lightning/loggers/comet.py [hunk 1]
```diff
@@ -301,6 +301,7 @@ def experiment(self) -> Union[CometExperiment, CometExistingExperiment, CometOff
                 self._experiment = CometOfflineExperiment(
                     offline_directory=self.save_dir, project_name=self._project_name, **self._kwargs
                 )
+            self._experiment.log_other("Created from", "pytorch-lightning")
```
Comment (akihironitta) [inline src/pytorch_lightning/loggers/comet.py:304]: Even though the behaviour from a user's perspective doesn't change at all, this should be documented in the changelog to notify the change to the community, IMO.

Comment (alexkuzmik) [reply src/pytorch_lightning/loggers/comet.py:304]: Is it something from your side or I should provide the description somewhere?

Comment (awaelchli) [reply src/pytorch_lightning/loggers/comet.py:304]: What's the best way to describe this @alexkuzmik? 

> Added:
> - The `CometLogger` now records the usage of Lightning in the experiment database

Something like that?

Comment (awaelchli) [reply src/pytorch_lightning/loggers/comet.py:304]: @alexkuzmik If you like you can add the description directly to src/pytorch_lightning/CHANGELOG.md in the section "Added".

Comment (alexkuzmik) [reply src/pytorch_lightning/loggers/comet.py:304]: @awaelchli 
"The CometLogger now flags the Comet Experiments as being created from Pytorch-Lightning for analytics purposes"

Comment (awaelchli) [reply src/pytorch_lightning/loggers/comet.py:304]: Looks great!

**Review by akihironitta — APPROVED** [review]

**Pull request merged**